e2e/pages/events/addHearingBookingDetailsEvent.page.js

```diff
@@ -31,24 +31,25 @@ module.exports = {
 
   addHearing() {
     I.click(this.addHearingButton);
+    I.wait(1);
```

Comment:
Is this required? Couldn't we way for something to appear, it's bad practice to wait 1 sec.